tests/data/GlyphsFileFormatv3.diff

```diff
@@ -0,0 +1,1679 @@
+  {
+  .appVersion = "3179";
+  .formatVersion = 3;
+  DisplayStrings = (
+  "/Smily",
+  "اا",
+  "/_part.test",
+  A,
+  B,
+  "ا",
+  A,
+  D
+  );
+  axes = (
+  {
+  hidden = 1;
+  name = Weight;
+  tag = wght;
+  }
+  );
+  classes = (
+  {
+  code = "A B C D E F G H I J K L M N O P Q R S T U V W X Y Z
+  ";
+  name = Uppercase;
++ notes = "a class note.";
+  },
+  {
+  automatic = 1;
+  code = "A B C D";
+  disabled = 1;
+  name = AllLetters;
+  }
+  );
+  customParameters = (
+  {
+  name = "Import Font";
+  value = {
+  Path = "files/LinkedFontv3.glyphs";
+  };
+  },
+  {
+  name = "Color Palettes";
+  value = (
+  (
++ (255,0,0,255),
```

Comment:
How glyphsLib is doing it matches the Markdown description. Not sure why Glyphs produced it this way.

https://github.com/schriftgestalt/GlyphsSDK/blob/Glyphs3/GlyphsFileFormat/GlyphsFileFormatv3.md